src/pudl/output_diff/cli.py

```diff
@@ -191,11 +191,15 @@ def __init__(self, root_path: str, gcs_project=None):
 
     def list_files(self) -> dict[str, str]:
         """Returns dict mapping from basenames to full paths."""
-        return {
-            os.path.basename(fpath): fpath
-            for fpath in self.fs.glob(self.root_path + "/*")
-            if self.match_filetype(fpath)
-        }
+        # TODO(rousik): check if root_path is an actual directory.
+        if self.root_path.endswith(".sqlite"):
+            return {"sqlite": self.root_path}
+        else:
+            return {
+                os.path.basename(fpath): fpath
+                for fpath in self.fs.glob(self.root_path + "/*")
+                if self.match_filetype(fpath)
+            }
```

Comment:
We've been trying to stick to using the newer `pathlib` interface for paths across the project. Are there reasons why this depends on `os.path`?